pkg/volume/projected/projected.go

```diff
@@ -328,6 +328,13 @@ func (s *projectedVolumeMounter) collectData() (map[string]volumeutil.FileProjec
 			}
 			tp := source.ServiceAccountToken
 
+			// When FsGroup is set, we depend on SetVolumeOwnership to
+			// change from 0600 to 0640.
+			mode := *s.source.DefaultMode
```

Comment:
This dereference is vaguely scary, but looks fine since we check nil above.

Reply:
added a test case for DefaultMode == nil